src/fairseq2/models/wav2vec2/model.py

```diff
@@ -113,10 +113,15 @@ def __init__(
         self.logit_temp = logit_temp
         self.diversity_loss_weight = diversity_loss_weight
 
-    def forward(self, batch: SequenceBatch) -> Wav2Vec2Output:
-        """
+    def extract_features(self, batch: SequenceBatch) -> Wav2Vec2Features:
+        """Extract features from the input sequences.
```

Comment:
very nit: Please have a blank line before `:param:` for sphinx to parse it correctly.